pkg/minikube/download/image.go

```diff
@@ -224,6 +226,35 @@ func ImageToCache(img string) error {
 	}
 }
 
+func ImageFromGitHubToCache(img, imgVersion string) (string, error) {
```

Comment:
to not be confused with github packages or githbub image registyr, we should name it that is CLEAR that we are downloading a TAR ball and then converting it to Image (which shoudl be last resort becasue it is much slower)

maybe add in the comment that the USER should consdier this in Last effort